tests/php/ORM/DataListTest.php

```diff
@@ -203,6 +214,330 @@ public function testSubtractWithCachedQuery(): void
         $this->assertSame(0, $queryCounter->getCount());
     }
 
+    public static function provideExcludeByList(): array
+    {
+        return [
+            'exclude by the whole list' => [
+                'list1Class' => ParentModel::class,
+                'list2Spec' => [
+                    'class' => ParentModel::class,
+                    'pre-filters' => [],
+                ],
+                'fieldToFilterBy' => 'ID',
+                'fieldInOtherList' => 'ID',
```

Comment:
```suggestion
                'fieldFromOtherList' => 'ID',
```
Should be able to do a quick find and replace

Reply:
Done